examples/complete/variables.tf

```diff
@@ -16,14 +16,32 @@ variable "resource_tags" {
   default     = []
 }
 
+# region needs to provide cross region support.
 variable "region" {
   description = "Region where resources will be created"
   type        = string
-  default     = "eu-gb"
+  default     = "us-south"
+}
+
+variable "cross_region_location" {
+  description = "Specify the cross-regional bucket location. Supported values are 'us', 'eu', and 'ap'."
+  type        = string
+  default     = "us"
+
+  validation {
+    condition     = var.cross_region_location == null || can(regex("us|eu|ap", var.cross_region_location))
+    error_message = "Variable 'cross_region_location' must be 'us' or 'eu', 'ap', or 'null'."
+  }
 }
 
 variable "resource_group" {
   type        = string
   description = "An existing resource group name to use for this example, if unset a new resource group will be created"
   default     = null
 }
+
+variable "activity_tracker_crn" {
+  type        = string
+  description = "Activity tracker crn for COS bucket. Using permanent instance as region is hardcoded in tests/pr_test.go."
+  default     = "crn:v1:bluemix:public:logdnaat:eu-de:a/abac0df06b644a9cabc6e44f55b3880e:b1ef3365-dfbf-4d8f-8ac8-75f4f84d6f4a::"
```

Comment:
default this null (meaning a new AT instance will be created by the example). In the test will will pass a value for this